### PR TITLE
Fix #87

### LINKED ISSUE #87: Unable to build with firebase as import
I'm unable to build a project that has firebase as an import.  It looks like some of the build steps are preprocessing imported modules in a way that's breaking uglify.

Environment:

  * macOS Sierra 
  * firebase 4.1.1
  * node v7.10.0
  * npm 4.6.1
  * yarn 0.24.5
  * preact-cli 1.1.0
  * preact 8.1.0
  * uglify-js 3.0.14

Reproduction steps:
```shell
$ preact create sample
$ cd sample
$ yarn add firebase
$ edit src/index.js # Add: import firebase from 'firebase';
$ yarn run build
  yarn run v0.24.5
  $ preact build
  bundle.js from UglifyJs
  Unexpected token: punc ()) [bundle.js:5468,33]
  ✨  Done in 6.07s.
```
Here's a [copy of the bundle.js](https://gist.github.com/jmadler/b9e947f1a95e2737161183646ffdd9db#file-bundle-js-L5470).  Note the ```()``` on line 5470 around char 33.

CC @addyosmani @jshcrowthe

### PR DESCRIPTION
**Pull request by lukeed**
Created a new [`webpack-plugin-replace`](https://github.com/lukeed/webpack-plugin-replace) in order to move forward & close #87.

I can add & update this plugin as needed 👍 

As it is, I went through the reproduction steps & it has been fixed.

### CODE REVIEW

File: src/lib/webpack-config.js [hunk 1]
```diff
@@ -378,11 +378,13 @@ const production = config => addPlugins([
...
-	new ReplacePlugin([{
-		// this is actually the property name https://github.com/kimhou/replace-bundle-webpack-plugin/issues/1
-		partten: /throw\s+(new\s+)?(Type|Reference)?Error\s*\(/g,
-		replacement: () => 'return;('
-	}]),
+	new ReplacePlugin({
+		include: /babel-helper$/,
+		patterns: [{
+			regex: /throw\s+(new\s+)?(Type|Reference)?Error\s*\(/g,
+			value: 'return;('
```
Comment (developit) [inline src/lib/webpack-config.js]: as per slack, we should make sure to retain the offset here via space padding:

```js
value: s => `return;${new Array(s.length-8).join(' ')}(`
```

**Comment by lukeed** [comment]
Hmm... `lukeed dismissed rkostrzewski’s stale review` that automatic wording seems a bit rough! 

Updated with @developit's suggestion for source-mapping (via character length).

**Comment by lukeed** [comment]
@jmadler 😆 I giggled. Glad to hear!

@developit Your lead, good sir -- I'd feel strange merging own PR.

**Comment by gokulkrishh** [comment]
@developit Can I see the PR is merged sometime back only 👍 Still asking anyways when can we expect new fixes for firebase in preact-cli ?. Working on an existing app porting from react to preact ;)

**Comment by developit** [comment]
@gokulkrishh today!

**Comment by gokulkrishh** [comment]
@developit Cool. Thanks 😃

**Pull request merged**